src/AbpHelper.Core/Models/DtoInfo.cs

```diff
@@ -2,15 +2,17 @@
 {
     public class DtoInfo
     {
-        public DtoInfo(string readTypeName, string createTypeName, string updateTypeName)
+        public DtoInfo(string readTypeName, string createTypeName, string updateTypeName, string getTypeName)
         {
             ReadTypeName = readTypeName;
             CreateTypeName = createTypeName;
             UpdateTypeName = updateTypeName;
+            GetTypeName = getTypeName;
```

Comment:
What about `GetListTypeName`?

Reply:
i wanted to fitler by custom field like this
public class GetBoaCommentDto : PagedAndSortedResultRequestDto
{
    public string Filter { get; set; }
    public Guid? TenantId { get; set; }
    public Guid BookOfAccountId { get; set; }
}
then decided to create this file automatically and then edit

Reply:
Yep, I understand. The thing is, The name `GetTypeName` misleads people since it is a DTO for getting a list but not getting a single.